README.md

```diff
@@ -79,6 +86,46 @@ from {{ metrics.calculate(
 
 `start_date` and `end_date` are optional. When not provided, the spine will span all dates from oldest to newest in the metric's dataset. This default is likely to be correct in most cases, but you can use the arguments to either narrow the resulting table or expand it (e.g. if there was no new customers until 3 January but you want to include the first two days as well). Both values are inclusive.
 
+### Migration from metric to calculate
+In version `0.3.0` of the dbt_metrics package, the name of the main macro was changed from `metric` to `calculate`. This was done in order to better reflect the work being performed by the macro and match the semantic naming followed by the rest of the macros in the package (describing the action, not the output). Additionally, the `metric_name` input was changed to take a single `metric` function or multiple `metric` functions provided in a list.
+
+To correctly change this syntax, you must:
+- change `metrics.metric` to `metrics.calculate`.
+- change `metric_name` to `metric('name_here')` 
+  - alternatively use `[metric('name_here'),metric('another_name_here')]` for multiple metrics
+
+## Develop
+There are times when you want to test what a metric might look like before defining it in your project. In these cases you should use the `develop` metric, which allows you to provide a single metric in a contained yml in order to simulate what the metric might loook like if defined in your project.
```

Comment:
>There are times
When? is this something that we expect most users to engage with? Or just something that we need to make available for integration developers? If the latter, it's OK for us to just say something like "this is a convenience function for integration developers, everyone else please ignore"

Reply:
I expect this to be used by both analytics engineers and integration developers. AE's can use this to test out metrics and integration developers can use it to develop interfaces to testing out new metrics.